c10/core/QScheme.h

```diff
@@ -16,13 +16,15 @@ enum class QScheme : uint8_t {
   PER_CHANNEL_AFFINE = 1,
   PER_TENSOR_SYMMETRIC = 2,
   PER_CHANNEL_SYMMETRIC = 3,
-  COMPILE_TIME_NUM_QSCHEMES = 4,
+  PER_CHANNEL_AFFINE_FLOAT_QPARAMS = 4,
+  COMPILE_TIME_NUM_QSCHEMES = 5,
```

Comment:
just curious, what is `COMPILE_TIME_NUM_QSCHEMES`? is there a reason it has to be last?

Reply:
i think it denotes the number of qschemes we have